### PR TITLE
Central working v2

### PR DESCRIPTION
**Pull request by omonk**
## Description - [Trello ticket](https://trello.com/c/2W01i4Ib/539-case-study-central-working-groomed)

- Add new case study for central working
- Move several methods and components from DoctorLink case study into common components

### CODE REVIEW

File: src/pages/case-study/central-working-changing-how-you-work.js [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+
+const Block3Col = styled(Col)`
+  padding-top: ${({ theme }) => theme.space[3]};
+`
+
+const Block4ImageCol = styled(Col)`
+  padding-top: ${({ theme }) => theme.space[4]};
+`
+
+const Block6ImageCol = styled(Col)`
```
Comment (GBouffard) [inline src/pages/case-study/central-working-changing-how-you-work.js]: Block4ImageCol & Block6ImageCol are the same. I would use a single component and think of another name

File: src/pages/case-study/central-working-changing-how-you-work.js [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+  padding-top: ${({ theme }) => theme.space[4]};
+`
+
+const Block7ImageCol = styled(Col)`
```
Comment (GBouffard) [inline src/pages/case-study/central-working-changing-how-you-work.js]: You can use the new name as above and use `width={[]}` prop with a custom array to render or not depending on breakpoints

File: src/pages/case-study/central-working-changing-how-you-work.js [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+  ${breakpoint('smallPhone', 'phone')`
+    padding-top: ${({ theme }) => theme.space[4]};
+  `}
+`
+
+const Block9ImageCol = styled(Col)`
```
Comment (GBouffard) [inline src/pages/case-study/central-working-changing-how-you-work.js]: Same comment as above. You can show or not with a custom `width={[]}`

File: src/pages/case-study/central-working-changing-how-you-work.js [hunk 4]
```diff
@@ -0,0 +1,385 @@
...
+  ${breakpoint('smallPhone')`
+      padding-top: ${({ theme }) => theme.space[4]}
+  `}
+`
+
+const StyledHr = styled(Hr)`
+  margin-left: auto;
+  margin-right: auto;
+
```
Comment (GBouffard) [inline src/pages/case-study/central-working-changing-how-you-work.js]: Does this work with `margin: 0 auto`; ? Nicer if it does. Dont know if some margins top bottom are set on Hr

File: src/components/Common/TanBackground.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components'
+
+export default styled.section`
+  background: ${props => props.theme.colors.tanBg};
```
Comment (GBouffard) [inline src/components/Common/TanBackground.js]: I would use `${({ theme }) => theme.colors.tanBg` for consistency with the rest of the code. It's not a blocker to merge. I like consistency :)

File: src/pages/case-study/central-working-changing-how-you-work.js (hunk 4 above)
Comment (omonk) [reply src/pages/case-study/central-working-changing-how-you-work.js]: Hr has margin top and padding bottom, this is why I've set it here explicitly

File: src/pages/case-study/central-working-changing-how-you-work.js (hunk 1 above)
Comment (omonk) [reply src/pages/case-study/central-working-changing-how-you-work.js]: I really don't think its necessary to create a shared component here. a `DRY` approach is not applicable in this situation

File: src/pages/case-study/central-working-changing-how-you-work.js (hunk 2 above)
Comment (omonk) [reply src/pages/case-study/central-working-changing-how-you-work.js]: This is an image column, I want it to display at all viewpoints but just give it some padding top on mobile down viewsa

File: src/pages/case-study/central-working-changing-how-you-work.js (hunk 3 above)
Comment (omonk) [reply src/pages/case-study/central-working-changing-how-you-work.js]: same reply as comment above 😉

File: src/components/Common/TanBackground.js (hunk 5 above)
Comment (omonk) [reply src/components/Common/TanBackground.js]: updated and the bluebackground component, nice one!

File: src/components/Common/CaseStudyCards/CaseStudyBlocks.js [hunk 6]
```diff
@@ -183,7 +183,61 @@ const TextAndResizedImageBlock = ({ data: { text, image } }) => (
...
+  padding-top: ${({ theme, desktop }) =>
+    desktop && desktop.top ? theme.space[desktop.top] : null};
+  padding-bottom: ${({ theme, desktop }) =>
+    desktop && desktop.bottom ? theme.space[desktop.bottom] : null};
+`}
+`
+
+const getImage = (blockImages, index) => blockImages && blockImages[index]
+
+const normaliseAll = (arr = []) =>
```
Comment (SemraSh) [inline src/components/Common/CaseStudyCards/CaseStudyBlocks.js]: I think we can should use more specific names than `arr` , `genericContents` maybe?

Comment (omonk) [reply src/components/Common/CaseStudyCards/CaseStudyBlocks.js]: Using `arr` here just refers to the argument name, what we get out is a an array of generic blocks

File: src/pages/case-study/central-working-changing-how-you-work.js (hunk 1 above)
Comment (GBouffard) [reply src/pages/case-study/central-working-changing-how-you-work.js]: ok. Why? Just trying to understand.

File: src/pages/case-study/central-working-changing-how-you-work.js (hunk 2 above)
Comment (GBouffard) [reply src/pages/case-study/central-working-changing-how-you-work.js]: ok. If that's the reason we can leave that 👍

File: src/pages/case-study/central-working-changing-how-you-work.js (hunk 4 above)
Comment (GBouffard) [reply src/pages/case-study/central-working-changing-how-you-work.js]: Alright. Makes sense then. I was too lazy to check 😆

File: src/components/Common/CaseStudyCards/CaseStudyBlocks.js (hunk 6 above)
Comment (SemraSh) [reply src/components/Common/CaseStudyCards/CaseStudyBlocks.js]: Yeah I understand it represents an array of something and it is an argument, I just think we should be more specific about naming. Up to you.

**Comment by SemraSh** [comment]
LGTM 👍

**Pull request merged**